tests/unit/plugins/modules/web_infrastructure/test_jenkins_build.py

```diff
@@ -93,6 +133,41 @@ def test_module_create_build(self, jenkins_connection, test_deps):
             })
             jenkins_build.main()
 
+    @patch('ansible_collections.community.general.plugins.modules.web_infrastructure.jenkins_build.test_dependencies')
+    @patch('ansible_collections.community.general.plugins.modules.web_infrastructure.jenkins_build.JenkinsBuild.get_jenkins_connection')
+    def test_module_stop_build(self, jenkins_connection, test_deps):
+        test_deps.return_value = None
+        jenkins_connection.return_value = JenkinsMock()
+
+        with self.assertRaises(AnsibleExitJson):
+            set_module_args({
+                "name": "host-check",
+                "build_number": "1234",
+                "state": "stopped",
+                "user": "abc",
+                "token": "xyz"
+            })
+            jenkins_build.main()
```

Comment:
(Looks like I forgot to add this...)

Can you add an assert for `changed=True` here?